### ISSUE TITLE
Add options to choose background color for an announcement 

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/food-oasis#2502]
### Overview
We need to change the background color on announcements to differentiate severity of the message.

#### Details
Currently there is no option to choose background color for the announcements. The pale yellow get added to the background of an announcement by default. In this issue we will add other color options that can be added as background color. We will use MUI default colors to accomplish that.

### Action Items
- [ ] Write pseudocode for how this feature will be implemented.
- [ ] Add the UI (using MUI)
- [ ] Add a failing test as per TDD (Test Driven Development)
- [ ] Add code for the implementation
- [ ] Test it in local before creating a PR

### Resources/Instructions
- Related file: https://github.com/hackforla/food-oasis/blob/develop/client/src/components/Admin/Announcements.jsx
